packages/webapp/src/containers/Filter/Tasks/index.jsx

```diff
@@ -29,6 +30,9 @@ const TasksFilterPage = ({ onGoBack }) => {
   const tasks = useSelector(tasksSelector);
   const dispatch = useDispatch();
   const locations = useSelector(locationsSelector);
+  const activeUsers = useSelector(userFarmsByFarmSelector)
+    .filter((user) => user.role != 'Owner')
+    .filter((user) => user.status != 'Inactive');
```

Comment:
Are owners not considered active users?

Reply:
You are correct about the owners, they should be included and I've made that change in the latest commit.

As for the "Unassigned" option, I will let Steve respond.